raiden-dapp/tests/unit/views/disclaimer-route.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { mount, Wrapper } from '@vue/test-utils';
+import flushPromises from 'flush-promises';
+import Vuex from 'vuex';
+import Vuetify from 'vuetify';
+import DisclaimerRoute from '@/views/DisclaimerRoute.vue';
+import ActionButton from '@/components/ActionButton.vue';
```

Comment:
Yeah, this is kinda stupid. I also don't understand it for sure. But I think it is probably because this is on a programmatic abstraction level. So sure in the browser this component just invisibly contains the other. So a click on it is basically a click on the button. But after all the event listener for the click is not on the div surrounding the button (plus eventually more stuff around by Vuetify), but the `input`/`button` HTML element in the DOM. I guess it would work if there would be an `@click` on the component itself, which isn't.
After all the only thing I won by this `findComponent`, that I'm independent of the exact component implementation (in fact the class name).

Reply:
Talked about it in person.